src/net.h

```diff
@@ -506,7 +506,12 @@ class CNode
         // Don't relay addr messages to peers that we connect to as block-relay-only
         // peers (to prevent adversaries from inferring these links from addr
         // traffic).
-        return m_conn_type != ConnectionType::BLOCK_RELAY;
+        if (m_conn_type == ConnectionType::BLOCK_RELAY) return false;
+
+        // Don't relay addr messages to peers that have sent disabletx, as they are
```

Comment:
Should the BIP recommendation around addr-relay updated to scope `GETADDR` ? Behavior should already be compliant on sender-side, we don't send getaddr to block-relay peers.